PC02_Grafitti_Andrade_Hrusovsky.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Erika Andrade
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+up() #move to eye
+goto(-17,115) # draws on left eye
+down()
+color("red")
+dot(20) #red eye on left
+up()
+goto(45,125) #draws on right eye
+dot(20) #red eye on right
```

Comment:
add turtle to the code
```suggestion
eyes.up() #move to eye
eyes.goto(-17,115) # draws on left eye
eyes.down()
eyes.color("red")
eyes.dot(20) #red eye on left
eyes.up()
eyes.goto(45,125) #draws on right eye
eyes.dot(20) #red eye on right
```